app/models/knight.rb

```diff
@@ -8,5 +8,20 @@ def image
     end
   end
 
+  def valid_move?(new_x, new_y)
+    on_board?(new_x, new_y) && (
+      allowed_deltas = [1, 2]
+      coerced_delta_x = allowed_deltas.delete((x_cord - new_x).abs)
+      coerced_delta_y = allowed_deltas.delete((y_cord - new_y).abs)
+      !!(coerced_delta_x && coerced_delta_y)
+    )
```

Comment:
Maybe we can refactor this code so we make a new method like so...
`def allowed_delta?(new_x, new_y)`
`allowed_deltas = [1, 2]`
`coerced_delta_x = allowed_deltas.delete((x_cord - new_x).abs)`
`coerced_delta_y = allowed_deltas.delete((y_cord - new_y).abs)`
`!!(coerced_delta_x && coerced_delta_y)`
`end`
and then on valid_move? we could change it like so...
`def valid_move?(new_x, new_y)`
`on_board?(new_x, new_y) && allowed_deltas?(new_x, new_y)
`end`
Something like this would be cleaner and would eliminate some potential future dependencies :)